apps/api/src/app/environments-v1/usecases/output-renderers/render-email-output.usecase.ts

```diff
@@ -14,8 +15,13 @@ export class RenderEmailOutputUsecase {
 
   @InstrumentUsecase()
   async execute(renderCommand: RenderEmailOutputCommand): Promise<EmailRenderOutput> {
-    const { emailEditor, subject } = EmailStepControlZodSchema.parse(renderCommand.controlValues);
-    const expandedSchema = this.transformForAndShowLogic(emailEditor, renderCommand.fullPayloadForRender);
+    const { body, subject } = EmailStepControlZodSchema.parse(renderCommand.controlValues);
```

Comment:
we will have regression here because people will already have stored in the DB emailEditor in the control value. or did we fix it somewhere else?

Reply:
We haven't released email in production!